rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -829,6 +831,13 @@ stream {
         {{ end }}
 
         location {{ $path }} {
+            {{ $ing := (getIngressInformation $location.Ingress $location.Path) }}
+            set $namespace      "{{ $ing.Namespace }}";
+            set $ingress_name   "{{ $ing.Rule }}";
+            set $service_name   "{{ $ing.Service }}";
+            set $service_port   "{{ $location.Port }}";
+            set $location_path  "{{ $location.Path }}";
```

Comment:
Could you describe a use case for this? Are `$request` or `$request_uri` not enough to identify which path got hit?